cmd/build.go

```diff
@@ -109,7 +110,7 @@ func runBuild() {
 		params = append(params, path.Join(repoPath, binary.Path))
 		err := sh.RunCommand("go", params...)
 		if err != nil {
-			fmt.Println("command failed:", err)
+			log.Fatal("command failed:", err, strings.Join(params, " "))
```

Comment:
`fatalMsg()` would be better in order to be consistent with the rest of the package, we don't use the log package anywhere else.

And TBH, I will probably change all error handling by using the `github.com/pkg/errors` package.